scripts/windows/build/Move-Image.ps1

```diff
@@ -31,3 +31,15 @@ Invoke-Expression $PushCommand
 if ($LASTEXITCODE) {
     throw "'$PushCommand' failed with exit code $LASTEXITCODE"
 }
+
+Write-Host "Cleaning up"
```

Comment:
I wouldn't add this to the script; it is management that we want to do on our custom agents so the code should only run on the agents (so, a task in the YAML). If a dev runs the script on their desktop then they may be surprised when the script wipes their docker images.